app/views/transfer/select.html.erb

```diff
@@ -0,0 +1,63 @@
+<%= content_for(:title, "Transfer Processing | MIT Libraries") %>
+
+<% content_for :additional_js do %>
+  <script src="https://cdn.datatables.net/1.10.24/js/jquery.dataTables.min.js"></script>
+<% end %>
+
+<link href="https://cdn.datatables.net/1.10.24/css/jquery.dataTables.min.css" rel="stylesheet">
```

Comment:
I wonder if our theme gem should allow `<% content_for :additional_css do %>` to better handle this use case?

(No action needed, just thinking out loud).

Reply:
I was on the point of asking why I couldn't find a layout in this app - didn't think to ask about the theme gem. Yes, I think that would be a good feature to add for a use case like this.

(I don't think that blocks this merging, though right? We can do that via maintenance?)